lib/Wireguard/WGmeta/Wrapper/Config.pm

```diff
@@ -1382,6 +1446,7 @@ Simple dumper method to print contents of C<< $self->{parsed_config} >>.
 
 =cut
 sub dump($self) {
+    use Data::Dumper;
```

Comment:
this will still get loaded at load time ... so you can as well put it at the top of the file ... else use 'require'

Reply:
method removed, the `parsed_config`-hash is available through the respective property anyways